Resources/Prototypes/_LostParadise/Entities/Objects/Specific/Robotic/borg_modules_syndi_sabotage.yml

```
@@ -0,0 +1,66 @@
+#- type: entity
+#  id: BaseBorgModuleSyndiMedical
+#  parent: BaseBorgModule
+#  abstract: true
+#  components:
+#  - type: Tag
+#    tags:
+#    - BorgModuleSyndicateMedical
+
+- type: entity
+  id: LPPBorgModuleChameleon
+  parent: [ LPPBaseBorgModuleSyndicateSabotage, BaseProviderBorgModule ] #BaseBorgModuleSyndiSabotage
```

yamllint (1.35.1)

[error] too many spaces inside brackets

(brackets)